src/PositionManager.sol

```diff
@@ -95,13 +95,17 @@ contract PositionManager is
 
     /// @param unlockData is an encoding of actions, params, and currencies
     /// @param deadline is the timestamp at which the unlockData will no longer be valid
-    function modifyLiquidities(bytes calldata unlockData, uint256 deadline)
```

Comment:
move natspec to IPoolManager and inheritdoc

note i think we should make the natspec really clear that `modifyLiquidities` is the more advanced integration style